### PR TITLE
Improve aeSwap integration

### LINKED ISSUE #935: Improve aeSwap integration
### Is your feature request related to a problem?

_No response_

### Describe the solution you'd like

Improve aeSwap integration

### Additional context

_No response_

### Epic

_No response_

### PR DESCRIPTION
**Pull request by redDwarf03**
- Improve aeSwap integration
- Move messaging to DApps list tab

### CODE REVIEW

**Comment by redDwarf03** [comment]
@Chralu TODO
- [ ] Take into account inactivity time constraints 
- [x] Cache retention issue (Base Hive) to be addressed
- [x] Mainnet pool list display crashes
- [ ] Certificate management
- [ ] Issue with network loss
- [ ] keyboard needs dot

**Comment by redDwarf03** [comment]
rebase fait

**Review by Chralu — COMMENTED** [review]
@redDwarf03 necessite `archethic-wallet-client-dart` de la PR suivante : https://github.com/archethic-foundation/archethic-wallet-client-dart/pull/48

**Review by Chralu — COMMENTED** [review]
@redDwarf03 je recontre le bug suivant sur Android API 28 :

https://github.com/pichillilorenzo/flutter_inappwebview/issues/2150

File: lib/infrastructure/repositories/dapps_repository.dart [hunk 1]
```diff
@@ -46,6 +46,12 @@ class DAppsRepositoryImpl implements DAppsRepositoryInterface {
   Future<List<DApp>> getDAppsFromNetwork(
     AvailableNetworks network,
   ) async {
+    return [
```
Comment (redDwarf03) [inline lib/infrastructure/repositories/dapps_repository.dart]: what a trap lol

**Comment by redDwarf03** [comment]
`flutter: Another exception was thrown: Failure.unknownAccount` error caused by

**Comment by redDwarf03** [comment]
tests ok with iOS (iphone & ipad)
ko with android
Issues created with AWCWebview label
The wallet code is ok but depends of flutterinappwebview lib

**Pull request merged**